changelogs/fragments/255_authorized_key_url.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
```suggestion
trivial:
```

Reply:
Thanks for creating fragments file! :)
Modifying docs usually specify as `trivial` instead of `minor_changes`. Therefore, could you change it to `trivial`?

Reply:
Sorry for the copy/paste error. Fixed to `trivial`.